pkgs/by-name/ti/tidgi/package.nix

```diff
@@ -0,0 +1,49 @@
+{ lib
+, stdenv
+, fetchurl
+, unzip
+}:
+
+stdenv.mkDerivation rec {
+  pname = "tidgi";
+  version = "0.9.6";
+
+  src =
+    if stdenv.isAarch64 then
+      fetchurl
+        {
+          url = "https://github.com/tiddly-gittly/TidGi-Desktop/releases/download/v${version}/TidGi-darwin-arm64-${version}.zip";
+          hash = "sha256-1Z9lxZZWrUVQEhBO/Kt2AS/uNs2XfihdL0iGrguPQ5g";
+        }
+    else
+      fetchurl
+        {
+          url = "https://github.com/tiddly-gittly/TidGi-Desktop/releases/download/v${version}/TidGi-darwin-x64-${version}.zip";
+          sha256 = "sha256-1mbj9dchgjmnfvrjh56ql99xbsq6n7kdr0fhww3j7rd4x63afjnm";
+        };
+
+  dontBuild = true;
+
+  nativeBuildInputs = [ unzip ];
+
+  sourceRoot = ".";
+
+  installPhase = ''
+    runHook preInstall
+
+    mkdir -p $out/Applications
+    cp -r *.app $out/Applications
```

Comment:
```suggestion
    mkdir -p "$out/Applications"
    cp -r *.app "$out/Applications"
```

Reply:
I don't think it would matter, useless the argument (in this case the directory path) contains space, but oh well, this is darwin.

Reply:
It's not like it's going to fail without it, but I think it's a good habit to surround paths